package.json

```diff
@@ -67,20 +67,20 @@
   },
   "devDependencies": {
     "@babel/core": "7.13.14",
-    "@rtsao/plugin-proposal-class-properties": "7.0.1-patch.1",
+    "@babel/plugin-proposal-class-properties": "7.13.0",
     "@babel/plugin-proposal-nullish-coalescing-operator": "7.13.8",
+    "@babel/plugin-proposal-object-rest-spread": "7.13.8",
     "@babel/plugin-proposal-optional-chaining": "7.13.12",
-    "@babel/plugin-syntax-dynamic-import": "7.8.3",
-    "@babel/plugin-proposal-class-properties": "7.13.0",
     "@babel/plugin-proposal-private-methods": "7.13.0",
-    "@babel/plugin-proposal-object-rest-spread": "7.13.8",
+    "@babel/plugin-syntax-dynamic-import": "7.8.3",
     "@babel/plugin-transform-react-constant-elements": "7.13.13",
```

Comment:
These looks like they are from my babel PRs? 

https://github.com/grafana/grafana/pull/32534
https://github.com/grafana/grafana/pull/32534

Reply:
I think they are artifacts of anything that touches package.json automatically.

I will remove them from this PR so you can update in the babel PRs